src/govuk/settings/_links.scss

```diff
@@ -4,43 +4,62 @@
 
 /// Enable new link styles
 ///
+/// If enabled, the link styles will change. Underlines will:
+///
+/// - be consistently thinner and a bit further away from the link text
+/// - have a clearer hover state, where the underline gets thicker to make the
+///   link stand out to users
+///
+/// These changes are an opt-in feature because you may:
+///
+/// - need to make changes to your service to introduce these new link styles
+///   consistently
+/// - want to avoid opting in if you have links within a multiple-column layout
+///   in your CSS - there is [a Chromium bug that affects such
+///   links](https://github.com/alphagov/govuk-frontend/issues/2204)
+///
+/// These new link styles will be enabled by default in a future major version
+/// of GOV.UK Frontend.
+///
 /// @type Boolean
 /// @access public
 
 $govuk-new-link-styles: false !default;
 
-/// Link underline thickness
+/// Thickness of link underlines
+///
+/// The default is whichever is calculated to be thicker:
```

Comment:
Bit of a GOV.UK style-related tweak needed here, because the bullets need to follow on from the lead-in sentence.

"The default will be either:
- 1px
- 0.0625rem, if it's thicker than 1px because the user has zoomed in their text"